apps/stata/stata_do_file_editor.talon

```diff
@@ -0,0 +1,25 @@
+# Commands for the Stata Do-File Editor
+os: windows
+app: stata
```

Comment:
app and win.title will need and before them here as well

Reply:
Please see my comment above. I believe Talon "ANDs" these.

Reply:
Yep, marked as resolved. fwiw I just filed https://github.com/talonhub/community/issues/1452

Reply:
However, I noticed that  for stata 15 the header did not match and improved it accordingly. Newer versions of stata  seem to match.